src/components/Authentication/SignIn.js

```diff
@@ -30,10 +30,12 @@ export default function SignIn() {
             email,
             password,
         }
-
+        console.log(user)
         //API call
+        console.log(process.env.REACT_APP_BASE_URL)
         axios.post(`${process.env.REACT_APP_BASE_URL}login/`, user)
             .then(res => {
+                console.log(res)
```

Comment:
idem

Reply:
@RamB98 please address this as well. Thank you!